frontend/src/container/TracesExplorer/TraceLoading/TraceLoading.tsx

```diff
@@ -0,0 +1,21 @@
+import './TraceLoading.styles.scss';
+
+import { Typography } from 'antd';
+import { useTranslation } from 'react-i18next';
+
+export function TracesLoading(): JSX.Element {
+	const { t } = useTranslation('trace');
```

Comment:
this seems like mostly copy of LogsLoading component. Can just pass one prop for the text here rather than duplicating the same?